test/ConsoleLogger.test.js

```diff
@@ -38,16 +38,19 @@ test.after(t => {
 test('ConsoleLogger should output results one by one followed by totals', async t => {
   const consoleLogger = new ConsoleLogger({objectMode: true});
   const passThrough = new PassThrough({objectMode: true});
-
-  let expectedData = serviceResults.slice(0);
   let count = 0;
+  let expectedData = serviceResults.slice(0);
+  const expectedSummary = '5 occurrences found in 2 projects';
```

Comment:
I think it gives a semantic context for what it's being used for - rather than just pushing a magic string on to the end of the array.